tests/Integration/DomainApiTest.php

```diff
@@ -0,0 +1,417 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+namespace Mailgun\Tests\Integration;
+
+use Mailgun\Tests\Api\TestCase;
+use Mailgun\Resource\Api\SimpleResponse;
+use Mailgun\Resource\Api\Domain\Credential;
+use Mailgun\Resource\Api\Domain\CredentialListResponse;
+use Mailgun\Resource\Api\Domain\DeliverySettingsResponse;
+use Mailgun\Resource\Api\Domain\DeliverySettingsUpdateResponse;
+use Mailgun\Resource\Api\Domain\SimpleDomain;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class DomainApiTest extends TestCase
+{
+    protected function getApiClass()
+    {
+        return 'Mailgun\Api\Domains';
+    }
+
+    /**
+     * Performs `GET /v3/domains` and ensures $this->testDomain exists
+     * in the returned list.
+     *
+     * @group domain
+     * @group integration
```

Comment:
I think this could be specified in phpunit.xml.dist. You could say that all tests in a directory are integration tests.